src/5e-SRD-Monsters.json

```diff
@@ -30267,7 +30557,66 @@
     "actions": [
       {
         "name": "Multiattack",
-        "desc": "The tarrasque can use its Frightful Presence. It then makes five attacks: one with its bite, two with its claws, one with its horns, and one with its tai l. It can use its Swallow instead of its bite."
+        "desc": "The tarrasque can use its Frightful Presence. It then makes five attacks: one with its bite, two with its claws, one with its horns, and one with its tail. It can use its Swallow instead of its bite.",
+        "options": {
+          "choose": 1,
+          "from": [
+            [
+              {
+                "name": "Frightful Presence",
+                "count": 1,
+                "type": "ability"
+              },
+              {
+                "name": "Bite",
+                "count": 1,
+                "type": "melee"
+              },
+              {
+                "name": "Claw",
+                "count": 2,
+                "type": "melee"
+              },
+              {
+                "name": "Horns",
+                "count": 1,
+                "type": "melee"
+              },
+              {
+                "name": "Tail",
+                "count": 1,
+                "type": "melee"
+              }
+            ],
+            [
+              {
+                "name": "Frightful Presence",
+                "count": 1,
+                "type": "ability"
+              },
+              {
+                "name": "Swallow",
```

Comment:
Is there a modifier on when it can use this? Is it worth adding it in a `note` field or is it clear enough? I'm not sure how much it matters.